templates/src/components/Pages/Pages.js

```diff
@@ -22,8 +22,24 @@ const Pages = ({ location, ...props }) => {
         <Transition appear key={location.pathname} timeout={getTransitionDuration(location.pathname)}>
           {state => (
             <Switch location={location}>
```

Comment:
I think you can wrap Suspense completely over the the Switch @DonghyukJacobJang 
```javascript
<Suspense fallback={noop}>     
     <Switch location={location}> 
        <Route exact path={routeKeys.Landing} component={<Landing transitionState={state} />} />
        <Route exact path={routeKeys.About} component={<About transitionState={state} /> }/>
        <Route component={NotFound} />
    </Switch>
</Suspense>```

Reply:
I can try that. Did not think of this idea. Thanks!